src/core/xfa/bind.js

```diff
@@ -635,6 +635,13 @@ class Binder {
             /* skipConsumed = */ this.emptyMerge
           ).next().value;
           if (!match) {
+            // If there is no match (no data) and occur.min === 0
+            // container is entirely excluded.
```

Comment:
There seem to be a couple of words missing in the middle, how about this instead:
```javascript
            // If there is no match (no data) and `occur.min === 0` then
            // the container is entirely excluded.
```